src/ESM.sol

```diff
@@ -3,6 +3,7 @@
 /// ESM.sol
 
 // Copyright (C) 2019-2021 Maker Ecosystem Growth Holdings, INC.
```

Comment:
Delete old copyright? Or are these supposed to stay?

Reply:
Yeah we should probably remove it and put from 2019 to the Dai Foundation.

Reply:
I'm not sure about that?  @brianmcmichael or @naszam do you know?

Reply:
The original copyright tag should stay since it was originally published under MEGH. We could add a new line to cover updates under Dai Foundation if we want.

Reply:
next line after that one is: `// Copyright (C) 2021-2022 Dai Foundation` so I think we are good then

Reply:
what about the following for now?
```
// Copyright (C) 2019-2020 Maker Ecosystem Growth Holdings, INC.
// Copyright (C) 2020-2022 Dai Foundation
```
ref. https://daifoundation.org/

Reply:
While we're at it, there's a copyright header in the `LICENSE` file that probably doesn't need to be there at all. The `LICENSE` file can be general.

Reply:
> what about the following for now?
> 
> ```
> // Copyright (C) 2019-2020 Maker Ecosystem Growth Holdings, INC.
> // Copyright (C) 2020-2022 Dai Foundation
> ```
> 
> ref. https://daifoundation.org/

It currently says:

```solidity
// Copyright (C) 2019-2021 Maker Ecosystem Growth Holdings, INC.
// Copyright (C) 2021-2022 Dai Foundation
```

@naszam are you saying we need to change the dates to 2020?

Reply:
not sure, I was referring the following:
```
In the beginning of 2020, the Maker Foundation transferred the ownership of Maker and Dai trademarks to the Dai Foundation for perpetual safekeeping.
```

Reply:
double checking I think it's fine as it is

Reply:
We had to replace MEGH licenses to DAI Foundation in the past, we should probably double check it with Soren.